From 7b175256bdb37b1c234e1f2742d010811a904cbb Mon Sep 17 00:00:00 2001
From: Daniel Dadap <ddadap@nvidia.com>
Date: Fri, 29 Sep 2023 09:37:47 -0500
Subject: [PATCH] Add libnvidia-gpucomp to the list of NVIDIA driver libraries

Upcoming versions of the NVIDIA driver will include a new component:

https://forums.developer.nvidia.com/t/new-driver-component-libnvidia-gpucomp/267060

Update the list of NVIDIA driver libraries so that it can be included
in the runtime environment along with the others.

Signed-off-by: Daniel Dadap <ddadap@nvidia.com>
---
 CHANGELOG.md       | 2 ++
 CONTRIBUTORS.md    | 1 +
 etc/nvliblist.conf | 1 +
 3 files changed, 4 insertions(+)

diff --git a/etc/nvliblist.conf b/etc/nvliblist.conf
index 27014d2eab..92c6803254 100644
--- a/etc/nvliblist.conf
+++ b/etc/nvliblist.conf
@@ -40,6 +40,7 @@ libnvidia-fbc.so
 libnvidia-glcore.so
 libnvidia-glsi.so
 libnvidia-glvkspirv.so
+libnvidia-gpucomp.so
 libnvidia-gtk2.so
 libnvidia-gtk3.so
 libnvidia-ifr.so
